Lock initial versions.

mheffner opened the pull request:
Lock versions for upcoming plugins install.

ehmicky commented:
Those are the versions that we pre-install in the buildbot or build-image, right?

What's the process to update those?

Also, plugin authors make PRs to submit their plugins: should they submit those fields as well? Should those versions be shown in a public repository?

mheffner commented:
> I am assuming those are the versions that we pre-install in the buildbot or build-image.

Yes, these will be the versions we preinstall.

> What would be the process to update those?

That is somewhat TBD, we talked about a review process to ensure that updates were reviewed. In either case, some type of PR to this repo would be the approach. We'll have some automation to rebuild/reinstall the latest versions, then a tag will be made for the "current" version.

> Also, plugin authors make PRs to submit their plugins: should they submit those fields as well

Yes, I think including the latest, known stable, version of their plugin would be appropriate.

mheffner commented:
Ok, rebased this since the latest updates.

@shortdiv At the moment I removed `netlify-plugin-predictive-prefetch` because there didn't appear to be an NPM package by that name. Is it named differently?
